JUnit4/src/test/java/StringUtilsTest.java

```diff
@@ -0,0 +1,77 @@
+package com.knoldus;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class StringUtilsTest {
+    StringUtils stringUtils;
+
+    @Before
+    public void setup(){
+        stringUtils=new StringUtils();
+    }
+
+    // testReverseString_Test1 method to test the expected reversedString.
+    @Test
+    public void testReverseString_Test1(){
```

Comment:
_Test1 _Test2 , why all methods with these suffixes 

_shouldReverse_givenCorrectInput -- like this, we need to have the name that reflects scenario as suffix.